package.json

```diff
@@ -1,6 +1,23 @@
 {
   "devDependencies": {
-    "@types/node": "^22.5.0",
-    "ts-node": "^10.9.2"
-  }
+    "@types/node": "22.5.0",
+    "ts-node": "10.9.2"
+  },
+  "name": "tsrun",
+  "version": "1.0.0",
+  "description": "TSRun is a wrapper script that guesses command fixes using the Gemini API.",
```

Comment:
I believe you generally write these descriptions as they should appear next to the title of the package (no need to repeat the name).

```suggestion
  "description": "A wrapper script that guesses command fixes using the Gemini API.",
```